index.js

```diff
@@ -139,8 +139,7 @@ class Conf {
 		let previousMigratedVersion = this._get(MIGRATION_KEY, '0.0.0');
 
 		const newerVersions = Object.keys(migrations)
-			.filter(candidateVersion => this._shouldPerformMigration(candidateVersion, previousMigratedVersion, versionToMigrate))
-			.sort(semver.compare);
```

Comment:
The sort was breaking because semver can't sort this array: ['>1', '>=2'] 😢 

Any suggestions?

Reply:
https://github.com/npm/node-semver#comparison has a method to use for sorting semver.

Add some tests to confirm it's working.

Reply:
What method are you refering to? Is it the same that we were already using? `semver.compare`? 
Because it does not work with ranges

Reply:
If we can't sort them, we shouldn't really sort at all then. Why do we even need to sort? The object key order should be enough.

Reply:
I guess so. 

However, sorting them would just allow the user to configure something like:

```js
const migrations = {
   '1.0.2': () => ....
   '0.0.1': () => ....
}
```

and it still would work in the correct order.


But it can be a non-issue, it's just harder and counter-intuitive to create migrations for older versions after a newer version migrate 🤔